sources/install/package_ad.sh

```diff
@@ -77,12 +77,12 @@ function install_crackmapexec() {
     # Source bc cme needs cargo PATH (rustc) -> aardwolf dep
     # TODO: Optimize so that the PATH is always up to date
     source /root/.zshrc || true
-    git -C /opt/tools/ clone https://github.com/Porchetta-Industries/CrackMapExec.git
+    git -C /opt/tools/ clone https://github.com/mpgn/CrackMapExec.git
```

Comment:
git -C /opt/tools/ clone --depth 1 https://github.com/mpgn/CrackMapExec.git

Reply:
Hey @gbe,

For crackmapexec we do not specify the depth because the user may need to pull (it's a specific case)

Reply:
Follow up from here @mpgn (https://github.com/ThePorgs/Exegol-images/pull/178) :
Please add --depth 1 as indicated @gbe ;)